src/bp/ui-studio/src/web/views/FlowBuilder/diagram/nodes/LinkWidget.tsx

```diff
@@ -108,7 +108,9 @@ class DeletableLinkWidget extends DefaultLinkWidget {
         onMouseEnter={() => this.setState({ selected: true })}
         onClick={() => {
           link.remove()
-          this.props.diagramEngine.repaintCanvas()
```

Comment:
I think we should keep this line, otherwise there's a small delay when you delete a link and it feels a bit "laggy"